src/assets/data/integrations/IntegrationList.js

```diff
@@ -0,0 +1,201 @@
+import docker from "./Images/docker.svg";
+import kubernetes from "./Images/kubernetes.svg";
+import aks from "./Images/aks.png";
+import eks from "./Images/eks.svg";
+import gke from "./Images/gke.png";
+import helm from "./Images/helm.svg";
+import kinD from "./Images/kinD.png";
+import kubesphere from "./Images/kubesphere.png";
+import minikube from "./Images/minikube.png";
+import openshift from "./Images/openshift.svg";
+import mac from "./Images/mac.svg";
+import homebrew from "./Images/homebrew.png";
+import wsl2 from "./Images/wsl2.svg";
+import raspberrypi from "./Images/raspberry-pi.svg";
+import traefik from "./Images/traefik-mesh.svg";
+import osm from "./Images/osm.svg";
+import nsm from "./Images/nsm.svg";
+import ngnix from "./Images/nginx-sm.svg";
+import linkerd from "./Images/linkerd.svg";
+import kuma from "./Images/kuma.svg";
+import istio from "./Images/istio.svg";
+import consul from "./Images/consul.svg";
+import cilium from "./Images/cilium.svg";
+import citrix from "./Images/citrix.svg";
+import appmesh from "./Images/app-mesh.svg";
+import tanzu from "./Images/tanzu.svg";
+import slack from "./Images/slack.svg";
+import github from "./Images/github.svg";
```

Comment:
Why add the images again?
That too locally in the code.
We have _almost_ all the images already present under `src/assets/images/service-mesh-icons`, if anything is missing then add it and import from there.

Reply:
done!

Reply:
Please note that all images need to be SVGs. If they aren't already available here, then please scour the net for them.

Reply:
See https://github.com/sahilsuman933/layer5/blob/c0c3917ce5fa3043b036503ec765885faecc2773/src/sections/Meshery/Meshery-platforms/supported-icons